homework-async-and-animations/src/helpers/scrollingChat.js

```diff
@@ -0,0 +1,8 @@
+const scrollingChat = function scroll() {
+  const chatWindow = document.querySelector('#chatWindow');
+  setTimeout(() => {
+    chatWindow.scrollTop = 1000;
+  }, 3000);
```

Comment:
magic numbers :)

Reply:
I redid a lot of things on the weekend =)))